source/backend/api/Areas/Leases/Controllers/LeasePaymentController.cs

```diff
@@ -64,7 +64,7 @@ public IActionResult AddPayment(long leaseId, [FromBody] PaymentModel paymentMod
         {
             _logger.LogInformation(
                 "Request received by Controller: {Controller}, Action: {ControllerAction}, User: {User}, DateTime: {DateTime}",
-                nameof(LeaseTermController),
+                nameof(LeasePeriodController),
```

Comment:
this should actually be **LeasePaymentController** - probably a copy/paste mistake on prior PRs